src/analytics/navigationTracker/navigationTracker.test.ts

```diff
@@ -1,6 +1,8 @@
 import { describe, expect, jest, test } from "@jest/globals";
 import { NavigationTracker } from "./navigationTracker";
 
+window.DI = { analyticsGa4: { cookie: { consent: true } } };
```

Comment:
Just to double check - we don't need to have a test for any of the trackers if the cookie consent is set to false? because for the form, navigation and page view trackers if the cookies is set to false - it will never be required to test?

Reply:
As unit test level, all the tests are covered within cookie.test.js and core.js. So it's not mandatory.
But because I also added a "guard" into the navigation tracking function (l.30), that is not a bad idea to check if the function return false (and stop to run) if consent === false.
I will add that also for other trackers.

Reply:
I pushed new unit tests called "Cookie Management".
Thanks for your feedback.